terms/confusionmatrix_df.tex

```diff
@@ -0,0 +1,199 @@
+
+These metrics are important in situations in which a particular type of misclassification, i.e. FP or FN, could have worse consequences than the other \citep{hassanien2017advances}. For example, FP are more tolerable than FN in classifiers which predict whether a patient has a disease or not. Even though both misclassifications are undesirable, in medical applications it's better to err on the side of caution since FN could be fatal.
+
+\textit{Accuracy} (ACC) is the proportion of correct predictions (Eq. \ref{eq:cm_acc}). It is a class-insensitive metric because it can give a high rating to a model which classifies majority class objects correctly but misclassifies interesting minority class objects \citep{branco2016survey}. The other metrics should be preferred since they're more class-sensitive and give better indicators when the dataset is imbalanced.
+
+\begin{equation}
+\label{eq:cm_acc}
+ACC = \frac{|TP \cup TN|}{|TP \cup FP \cup TN \cup FN|}
+\end{equation}
+
+\textit{Negative Predictive Value} (NPV) is the ratio of the correct negative predictions from the total negative predictions (Eq. \ref{eq:cm_npv}).
+
+\begin{equation}
+\label{eq:cm_npv}
+NPV = \frac{|TN|}{|TN \cup FN|}
+\end{equation}
+
+\textit{True Negative Rate} (TNR), or \textit{Specificity}, is the ratio of the correct negative predictions from the total true negatives (Eq. \ref{eq:cm_tnr}).
+
+\begin{equation}
+\label{eq:cm_tnr}
+TNR = \frac{|TN|}{|TN \cup FP|}
+\end{equation}
+
+\textit{True Positive Rate} (TPR), also called \textit{Sensitivity} or \textit{Recall}, is the ratio of the correct positive predictions from the total true positives (Eq.\ref{eq:cm_tpr}).
+
+\begin{equation}
+\label{eq:cm_tpr}
+TPR = \frac{|TP|}{|TP \cup FN|}
+\end{equation}
+
+Sensitivity and Specificity can be combined into a single metric (Eq. \ref{eq:cm_ss}). These metrics are often used in  domains in which minority classes are important \citep{kuhn2013applied}. For example, a medical classifier \citep{el2010hybrid} could be evaluated by measuring Sensitivity to find the number of patients with a condition who tested positive, and Specificity to measure the number of patients who don't have the condition and tested negative.
+
+\begin{equation}
+\label{eq:cm_ss}
+\textnormal{\textit{Sensitivity}} \times \textnormal{\textit{Specificity}} = \frac{|TP| \times |TN|}{|TP \cup FN| \times |TN \cup FP|}
+\end{equation}
+
+\textit{Positive Predictive Value} (PPV), or \textit{Precision}, is the ratio of the correct positive predictions from the total positive predictions (Eq. \ref{eq:cm_ppv}).
+
+\begin{equation}
+\label{eq:cm_ppv}
+PPV = \frac{|TP|}{|TP \cup FP|}
+\end{equation}
+
+Precision and Recall are borrowed from the discipline of \textit{Information Extraction} \citep{sokolova2009systematic}. A composite metric called \textit{F-score}, \textit{F1-score}, or \textit{F-measure} (Eq. \ref{eq:cm_fscore}) can be derived by finding their harmonic mean \citep{kelleher2015fundamentals}.
```

Comment:
There is a nice image of a dart board explaining precision and recall ... please add.

Reply:
The ones I found (and ended up adding to the doc) are about the difference between accuracy and precision.